app/src/main/java/com/pvp/app/ui/screen/decoration/Decoration.kt

```diff
@@ -140,7 +141,14 @@ fun DecorationCards(
                     }
                 }
             },
-            header = type.toString()
+            header = type.toString(),
+            modifier = Modifier
```

Comment:
Modifier became providable in the parameters for this component so that other places could adjust required style as wanted, it wouldn't be hard-coded.